src/grpc-server-streaming-method.js

```diff
@@ -19,6 +19,7 @@ class GrpcServerStreamingMethod extends GrpcMethod {
    * @return {void}
    */
   async exec (call) {
+    const responseMetadata = {}
```

Comment:
since data gets written to `responseMetadata`, can we make a note and change this to `let` to signify that the value is expected to change?

Reply:
I don't think I agree with that. `let` signifies that the name might be reassigned, which is the opposite of what we want here. I will make a comment about it though.